logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -142,8 +141,7 @@ sealed class ConversationDetails(open val conversation: Conversation) {
         val unreadMessagesCount: Long = 0L,
         val unreadMentionsCount: Long = 0L,
         val lastUnreadMessage: Message?,
-        val isSelfUserMember: Boolean = true,
-        val isSelfCreated: Boolean = false
+        val isSelfUserMember: Boolean = true
```

Comment:
Would this be really needed if we already have access to the same field within the `conversation` object?

Reply:
that's right, this is why i removed that field here :)

Reply:
But `isSelfUserMember` is still part of the `Conversation` data class in line 30 🤷

Reply:
after your question i noticed that both `isSelfUserMember` and `isSelfCreated ` should be not in `Conversation` but in `ConversationDetails.OneOne` and `ConversationDetails.Group` as these fields are related only to these data classes. So I moved them :)